device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json

```diff
@@ -0,0 +1,130 @@
+{
+    "chassis": {
+        "name": "7800R3A-36DM2-LC",
+        "components": [],
```

Comment:
"components": [
            {
                "name": "Aboot()"
            },
            {
                "name": "Scd(addr=0000:00:18.7)"
            },
            {
                "name": "Scd(addr=0000:08:00.0)"
            }
        ],

Reply:
apologies; I guess I had an outdated version of the patch. I just pushed a file that should be identical to master.